### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by AnhToof**
Init project with the default structure and remove .idea/ folder

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            android:allowBackup="true"
+            android:icon="@mipmap/ic_launcher"
+            android:label="@string/app_name"
+            android:roundIcon="@mipmap/ic_launcher_round"
+            android:supportsRtl="true"
+            android:theme="@style/AppTheme">
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN"/>
+
```
Comment (NguyenChanhTiep) [inline app/src/main/AndroidManifest.xml]: remove space

File: app/src/androidTest/java/com/lobesoftware/toof/firebase_chat_001/ExampleInstrumentedTest.kt [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.lobesoftware.toof.firebase_chat_001
+
+import android.support.test.InstrumentationRegistry
+import android.support.test.runner.AndroidJUnit4
+
```
Comment (onstonboy) [inline app/src/androidTest/java/com/lobesoftware/toof/firebase_chat_001/ExampleInstrumentedTest.kt]: remove blank line

File: app/src/main/res/drawable-v24/ic_launcher_foreground.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:aapt="http://schemas.android.com/aapt"
+        android:width="108dp"
```
Comment (NguyenChanhTiep) [inline app/src/main/res/drawable-v24/ic_launcher_foreground.xml]: move 108dp to folder dimens.xml
move dp and sp to file dimens.xml

File: app/src/main/res/drawable-v24/ic_launcher_foreground.xml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        android:height="108dp"
```
Comment (NguyenChanhTiep) [inline app/src/main/res/drawable-v24/ic_launcher_foreground.xml]: move 108dp to folder dimens.xml
move dp and sp to file dimens.xml

File: app/src/main/java/com/lobesoftware/toof/firebase_chat_001/MainActivity.kt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+package com.lobesoftware.toof.firebase_chat_001
+
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+
+class MainActivity : AppCompatActivity() {
+
```
Comment (onstonboy) [inline app/src/main/java/com/lobesoftware/toof/firebase_chat_001/MainActivity.kt]: remove blank line

File: app/src/main/res/drawable/ic_launcher_background.xml [hunk 6]
```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:height="108dp"
```
Comment (NguyenChanhTiep) [inline app/src/main/res/drawable/ic_launcher_background.xml]: move 108dp to folder dimens.xml
move dp and sp to file dimens.xml

File: app/src/main/res/drawable/ic_launcher_background.xml [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        android:width="108dp"
```
Comment (NguyenChanhTiep) [inline app/src/main/res/drawable/ic_launcher_background.xml]: move 108dp to folder dimens.xml
move dp and sp to file dimens.xml

File: app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".MainActivity">
+
+    <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Hello World!"
```
Comment (onstonboy) [inline app/src/main/res/layout/activity_main.xml]: Should be define in string.xml

File: app/src/main/res/values/dimens.xml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="dp_108">108dp</dimen>
+</resources>
```
Comment (truyenz) [inline app/src/main/res/values/dimens.xml]: add blank line

**Pull request merged**